app/assets/stylesheets/urls.scss

```diff
@@ -1,3 +1,15 @@
 // Place all the styles related to the Urls controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.hidden {
+  display: none;
+}
+
+.show {
+  display: block;
+}
+
+.colorize {
+  color: magenta;
```

Comment:
Color `magenta` should be written in hexadecimal form as `#ff00ff`<br>Color literals like `magenta` should only be used in variable declarations; they should be referred to via variable everywhere else.